src/components/ChecButton.vue

```diff
@@ -39,18 +39,21 @@ export default {
           'blue',
           'green',
           'red',
+          'purple',
+          'orange',
+          'white',
+          'gray',
         ].includes(color);
       },
       default: 'secondary',
     },
     /**
-     * The style variant of the button. Includes size, border radius, etc. One of 'large', 'regular', 'small',
-     * 'round', 'tag'
+     * The style variant of the button. Includes size, border radius, etc. One of 'regular', 'small', 'round', 'tag'
      */
     variant: {
       type: String,
       validate(variant) {
-        return ['large', 'regular', 'small', 'round', 'tag'].includes(variant);
+        return ['regular', 'small', 'round', 'text', 'tag'].includes(variant);
```

Comment:
Did you mean to remove large?

Is it used anywhere in the dashboard or auth service?

Reply:
Oh yeah, I forgot to add that to the description. It's not used, and has been removed from Figma.